src/components/Header.jsx

```diff
@@ -19,7 +22,26 @@ function Header() {
         <button className={style.btn}>Resources</button>
         <hr className={style.hr} />
         <button className={style.btn}>Login</button>
-        <button className={`${style.btn} ${style.btn_sign}`}>sign Up</button>
+        <button className={classNames}>Sign Up</button>
+      </div>
+      <div className={style.menu_big_screen}>
+        <nav>
+          <ul className={style.menu_big_screen}>
+            <li>
+              <a href="#">Features</a>
+            </li>
+            <li>
+              <a href="#">Pricing</a>
+            </li>
+            <li>
+              <a href="#">Resources</a>
+            </li>
+          </ul>
+        </nav>
+        <div>
+          <p>Login</p>
```

Comment:
Why logic is a `p` tag? Have you seen this as a user? I guess it has to be either button or anchor.

Reply:
oops